todo-list/main.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+'''
+    File name: main.py
+    Author: Henry Letton
+    Date created: 2020-08-14
+    Python Version: 3.7
+    Desciption: This program manages a user todo list, using CLI.
+'''
+
+# Import required modules
+from datetime import datetime # For dealing with due dates
+import pickle # For saving list to disc
+import os # For moving working directory
+
+# Make sure the wording directory is the same location as these (example below)
+# os.getcwd()
+# os.chdir('C:\\Users\\Henry\\OneDrive\\Documents\\Python\\Matts-Coding-Challenges\\todo-list')
+
+
+# Import functions and objects from other file
+from fn_and_obj import load_prev_list, save_list_to_disc, ToDoList, ToDoItem
+
+
+# New to do list or can use previous
+your_todo_list = load_prev_list()
+
+
+# Create while loop to ask for user input repeatably until the user is finished
+user_editing = True # Break variable, when set to false will end while loop
+
+while user_editing:
```

Comment:
Meta-comment, what you've done is fine for an input loop, but Python offers a kind of built in REPL that maybe could have been made use of - https://docs.python.org/3/library/cmd.html.